.github/workflows/sync-examples.yml

```
@@ -10,3 +15,76 @@ jobs:
     steps:
       - name: Checkout PGM workshop repository
         uses: actions/checkout@v4
+        with:
+          path: power-grid-model-workshop
+
+      - name: Checkout PGM Repository examples
+        uses: actions/checkout@v4
+        with:
+          repository: PowerGridModel/power-grid-model
+          ref: main
+          path: power-grid-model
+          sparse-checkout: docs/examples
+          sparse-checkout-cone-mode: false
+
+      - name: Display $GITHUB_WORKSPACE contents
+        run: |
+          ls $GITHUB_WORKSPACE
```

here and below: single line commands can be written like

```suggestion
        run: ls $GITHUB_WORKSPACE
```